src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -0,0 +1,173 @@
+@import '~amo/css/styles';
+
+.HeroRecommendation {
+  color: $white-100;
```

Comment:
It's the first time we use this variable, everywhere else we use `$white`, which is defined in https://github.com/mozilla/addons-frontend/blob/67faca626a21317490cdbc24c358f8e2d2d8dba0/src/ui/css/vars.scss#L5

Can we keep `$white` here, for consistency? I suppose we can change `#fff` to `$white-100` in the `vars.scss` file.